lib/resolve-deps.js

```diff
@@ -30,47 +36,47 @@ var getDepsForFile = function (file, extraImports) {
   if (extraImports && extraImports.length > 0) {
     matches = extraImports.concat(matches);
   }
+
   if (matches.length > 0) {
-    _.each(matches, function (importedFile) {
-      var importedPath = path.resolve(path.dirname(file), importedFile);
+    let importedFile = await _.each(matches);
+    let importedPath = path.resolve(path.dirname(file), importedFile);
       // recursively get dependencies for imported files
-      deps[file].push(getDepsForFile(importedPath));
-    });
+    let importedDeps = await getDepsForFile(importedPath);
+    deps[file].push(importedDeps);
   }
   return deps;
-};
+}
+
+async function buildScriptFromDeps (deps) {
+
+  let script = '';
+  let [subDepsArray, file] = await _.each(deps);
+  if (subDepsArray.length > 0) {
+    let subDeps = await _.each(subDepsArray);
+    script += await buildScriptFromDeps(subDeps);
+  }
+  let readFile = await fs.readFile(file).toString('utf8');
```

Comment:
If you want to still do this in one line, you can make sure to combine `await` and `fs.readFile` using parens:

```js
let readFile = (await fs.readFile(file)).toString('utf8');
```

But in this case, there's an easy way to do this because `readFile` takes a second parameter. You can just do:

```js
let readFile = await fs.readFile(file, 'utf8');
```

One final comment: I would probably name the variable something else, since `readFile` is the name of a function that reads a file. Maybe `fileContents`?